packages/mobx/src/types/actionannotation.ts

```diff
@@ -61,7 +62,7 @@ function make_(adm: ObservableObjectAdministration, key: PropertyKey): void {
     }
     if (annotated) {
         recordAnnotationApplied(adm, this, key)
-    } else if (!this.isDecorator_) {
+    } else if (!adm.target_[storedAnnotationsSymbol]) {
```

Comment:
Probably I misunderstand the logic, but should this be `adm.target_[storedAnnotationsSymbol]?.[key]`?

Reply:
Actually not a bad idea :) I think I was worried it would return false if `recordAnnotationApplied` is called before (it removes the annotation from `adm.target_[storedAnnotationsSymbol]`), but that can't happen here...